modules/indigo/module/inc/indigo/of_connection_manager.h

```diff
@@ -814,5 +814,15 @@ indigo_cxn_subbundle_set2(uint32_t num_subbundles,
                           indigo_cxn_subbundle_start_t starts[],
                           indigo_cxn_subbundle_finish_t finishes[]);
 
+/**
+ * Adjust the OpenFlow connection timeout in switch by changing the
+ * connection allowed outstanding count of keepalive messages.
```

Comment:
how about "Change the maximum number of outstanding keepalives, so that the connection timeout interval can be adjusted." ?